.circleci/config.yml

```diff
@@ -3,28 +3,21 @@
 version: 2
 jobs:
   test-1.13:
-    working_directory: /go/src/go.universe.tf/metallb
     docker:
       - image: cimg/go:1.13
     steps:
       - checkout
-      - run: sudo apt-get install python-pip
-      - run: sudo pip install invoke semver pyyaml
-      - run: echo "export GO111MODULE=on" >>$BASH_ENV
```

Comment:
Not sure I follow how the `pip` packages and the env var works on the CI without this. How does it work?

Reply:
We are not using `pip` (or Python in general) in this job so there is no reason to install the packages.
`go test` works fine without `echo "export GO111MODULE=on" >>$BASH_ENV` so looks like we don't need it. Less code == better IMO.

Reply:
Oh, I had misunderstood the diff. Sorry, my bad, now it all makes sense. Perfect, thanks! :)